resources/stylesheet.css

```diff
@@ -148,15 +148,51 @@ hr {
 	}
  /*Forum Helpers List*/
 
+/*Credits*/
+.credits_content {
+	color: black;
+	font-size: 20px;
+	width: 100%;
+	text-align: center;
+	margin-top: 0px;
+	font-family: Helvetica, Arial, sans-serif;
```

Comment:
One last thing: do we really want to use Helvetica?\Part of the site overhaul was removing other fonts and only having Arial in the body text.

Reply:
Both the main page and the list of Forum Helpers page have this listed as the font-family. I just copy and pasted from there. Looking at both of those pages now though, it looks like Helvetica isn't even being rendered. I think this should be filed as a separate issue since it affects more than one page.

Reply:
I see. Well, other than that everything looks good!